### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Lornakaboro**
## In this activity I did the following : 

- Created a method called current_user in my ApplicationController to make current user data available to all controllers
- Created forms to perform the following functions:

- [ ] Creates a Post on behalf of the current_user 
- [ ] Create a comment on behalf of the current_user

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,35 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
```
Comment (emmanuellekamwa) [inline app/controllers/posts_controller.rb:2]: - [ ] Please remove the empty methods present in this file. It doesn't look professional.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑⛔**
Hello @Lornakaboro 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Lornakaboro) [reply app/controllers/posts_controller.rb:2]: Hello @emmanuellekamwa If you check my code you'll see that you pointed to a method that has already been removed.

**Review by jerryowusu — APPROVED** [review]
Hi @Lornakaboro 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

good job


## Highlights
Nice design :+1: 
Nice commit messages :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**